src/main/java/com/google/api/codegen/transformer/ruby/RubyGapicSurfaceTransformer.java

```diff
@@ -187,16 +192,24 @@ private ViewModel generateApiClass(GapicInterfaceContext context, String package
     xapiClass.apiMethods(methods);
 
     xapiClass.toolkitVersion(GeneratorVersionProvider.getGeneratorVersion());
+
+    final boolean isLongRunning =
+        RubyUtil.isLongrunning(context.getProductConfig().getPackageName());
+
+    // use explicit gem name if provided in the configuration
+    Map<String, String> packageSettings =
```

Comment:
Note: We might be able to not have this ultra specific logic if we go with this solution.